build/init_and_run.sh

```diff
@@ -12,13 +12,25 @@ echo "INDY_NODE_IP=${INDY_NODE_IP:=0.0.0.0}"
 echo "INDY_NODE_PORT=${INDY_NODE_PORT:=9701}"
 echo "INDY_CLIENT_IP=${INDY_CLIENT_IP:=0.0.0.0}"
 echo "INDY_CLIENT_PORT=${INDY_CLIENT_PORT:=9702}"
-
 echo "INDY_NODE_SEED=[$(echo -n $INDY_NODE_SEED|wc -c) characters]"
+echo "INDY_NODE_LOG_LEVEL=${INDY_NODE_LOG_LEVEL:=0}"
+echo "INDY_NODE_LOG_STDOUT=${INDY_NODE_LOG_STDOUT:=False}"
```

Comment:
I suggest defaulting to STD output, since people might expect to find logs with `docker logs`.

```suggestion
echo "INDY_NODE_LOG_STDOUT=${INDY_NODE_LOG_STDOUT:=True}"
```